app/controllers/clearance/sessions_controller.rb

```diff
@@ -33,4 +37,8 @@ def url_after_create
   def url_after_destroy
     sign_in_url
   end
+
+  def redirect_url
```

Comment:
I'd also be cool with "url_for_users_already_signed_in" which defaults to `url_after_create`? Something like that. That gives maximum flexibility.